restapi/endpoints_test.go

```diff
@@ -145,7 +145,28 @@ func (suite *endpointTestSuite) TestReturnAllUniqueTags() {
 }
 
 func (suite *endpointTestSuite) TestGetAllStories() {
```

Comment:
Is it difficult to add a test for editing a story?

Reply:
It probably is. We are passing in formdata and also mocking uploading to s3 might be hard, but we can look into it. Would it make sense to look into testing after this gets merged in or right now?

Reply:
Yeah lets get this merged first